social_core/backends/azuread_b2c.py

```diff
@@ -72,8 +72,8 @@ class AzureADB2COAuth2(AzureADOAuth2):
     ]
 
     @property
-    def tenant_id(self):
-        return self.setting('TENANT_ID', 'common')
+    def tenant_name(self):
+        return self.setting('TENANT_NAME', 'common')
```

Comment:
This breaks setup for existing users, are these different, or this is just a different name for the same value?